images/public-log-vendor-matcher/pg-init.d/lib-15-stop-postgres.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Stop Postgres
```

Comment:
So it doesn't seem like postgres is actually used?  Can we use something other than pg-init.d as our entrypoint?  Like this appears at a glance to be a set of scripts that are invoked in order.  

I would rather see something like `/app/entrypoint.sh` that runs the scripts in the expected order

Reply:
Postgres is used for preparing the data from the ASN providers and trimming some data down, such as
- https://github.com/kubernetes/k8s.io/pull/2710/files#diff-66e7079932b4bc08f013e6db34207fdce92d36623328555e1362bc5ad167087eR1
- https://github.com/kubernetes/k8s.io/pull/2710/files#diff-338e9f84ded10fcacd04517dc8b9c9581dd05a17889263e520d666753f3381f3R1
- https://github.com/kubernetes/k8s.io/pull/2710/files#diff-30b2418800710c115d14c538080097c6a6f78ea0eaf9816f6fd006e11a983b0bR1

I like the idea of using the primitives that Postgres provides, with the init scripts. It works quite well

Reply:
as long as you can explain what this does, and as long as we run this someplace where an ephemeral container won't be accessible to others and potentially leak data, then I'm ok relying on what postgres provides out of the box